src/tokens/builder.rs

```diff
@@ -133,6 +133,31 @@ impl<'a> PasetoBuilder<'a> {
     self
   }
 
+  /// Sets multiple arbitrary claims by passing a HashMap of keys and values (keys inside the json token).
+  /// If a key in our passed map already exists in the internal claims, the value will be updated
+  /// with the value of the key in the HashMap we pass in this method.
+  /// For more info, reference the std::iter::Extend trait documentaton at
+  /// https://doc.rust-lang.org/std/iter/trait.Extend.html
```

Comment:
FWIW these links at the end aren't proper rustdoc links (they won't be linkified) since they're not enclosed in `<>` (since rustdoc is just markdown).

However, there's actually an even better thing we can do here which is just use: [`std::iter::Extend`], see: <https://doc.rust-lang.org/rustdoc/linking-to-items-by-name.html>.

As such a better way to write this may be:

```suggestion
  /// Provide multiple arbitrary claims at once, if a key is already present it will be updated.
  ///
  /// Mirrors [`std::iter::Extend`] for normal maps.
```

Reply:
Thanks for that.  I had no clue!